tests/Integration/BibleFileTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
This new integration test is to test the BibleFile model and when it should fetch the Book relationship for the API v4.